packages/stack/src/views/MaskedView.macos.tsx

```diff
@@ -0,0 +1,3 @@
+import StubbedMaskedView from './MaskedView';
```

Comment:
Wouldn't this resolve to `MaskedView.native.tsx`? Maybe we need to rename `MaskedView.tsx` to `MaskedViewStub.tsx` and then re-export it.

```js
export * from './MaskedViewStub';
```